File: docflow-backend/src/main/java/com/brutecx/docflow_backend/api/dto/audit/LifecycleDeniedAuditCursorPageDTO.java
package com.brutecx.docflow_backend.api.dto.audit;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

public record LifecycleDeniedAuditCursorPageDTO(
        List<LifecycleDeniedAuditDTO> items,
        boolean hasMore,
        Instant nextCursorTimestamp,
        UUID nextCursorId
) {
}

File: docflow-frontend/types/api/LifecycleDeniedAuditCursorPageDTO.ts
import { LifecycleDeniedAuditRow } from "./LifecycleDeniedAuditRow";

export type LifecycleDeniedAuditCursorPageDTO = {
  items: LifecycleDeniedAuditRow[];
  hasMore: boolean;
  nextCursorTimestamp?: string | null;
  nextCursorId?: string | null;
};
